packages/affine/block-surface/src/element-model/mindmap.ts

```diff
@@ -168,34 +168,11 @@ export class MindmapElementModel extends GfxGroupLikeElementModel<MindmapElement
       return;
     }
 
-    let splitDir: LayoutType | undefined = undefined;
     const tree = this._tree;
-    const splitPoint = tree.children.findIndex((child, index) => {
-      if (
-        child.detail.preferredDir === LayoutType.LEFT ||
-        (child.detail.preferredDir === LayoutType.RIGHT &&
-          child.children[index + 1]?.detail.preferredDir !== LayoutType.RIGHT)
-      ) {
-        splitDir = child.detail.preferredDir;
-        return true;
-      }
-
-      return false;
-    });
-
-    if (
-      splitPoint === -1 ||
-      (splitDir === LayoutType.LEFT && splitPoint >= tree.children.length / 2)
-    ) {
-      const mid = Math.ceil(tree.children.length / 2);
-
-      tree.right.push(...tree.children.slice(0, mid));
-      tree.left.push(...tree.children.slice(mid));
-    } else {
-      tree.right.push(...tree.children.slice(0, splitPoint + 1));
-      tree.left.push(...tree.children.slice(splitPoint + 1));
-    }
+    const splitPoint = Math.ceil(tree.children.length / 2);
```

Comment:
Not really related to this pr, but is it possible now to move this model to `affine-model`?

Reply:
@doodlewind seems to be working on this.